src/porepy/numerics/fv/fv_elliptic.py

```diff
@@ -44,34 +44,39 @@ def extract_pressure(self, g, solution_array, d):
             g (grid): To which the solution array belongs.
             solution_array (np.array): Solution for this grid obtained from
                 either a mono-dimensional or a mixed-dimensional problem.
-            d (dictionary): Data dictionary associated with the grid. Not used,
+            data (dictionary): Data dictionary associated with the grid. Not used,
                 but included for consistency reasons.
         Returns:
             np.array (g.num_cells): Pressure solution vector. Will be identical
                 to solution_array.
         """
         return solution_array
 
-    def extract_flux(self, g, solution_array, d):
+    def extract_flux(self, g, solution_array, data):
```

Comment:
Be aware of the limitations of this function: It does not account correctly for inter-dimensional fluxes, and should probably only be used on single-grid problems.

Reply:
ok, this is a general comment. I resolve it